### DISCUSSION TITLE
auto type specifier is a c++11 extension wc++11 extensions

### DISCUSSION THREAD

**Discussion by wotterfree** [clangd/clangd#1218] [Q&A]
## envs:
- Apple clang version 13.1.6 (clang-1316.0.21.2.5)
- Target: arm64-apple-darwin21.5.0
- Thread model: posix
- InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
## question:🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻
I have encountered some problems in vscode. After installing the clangd plugin, it always prompts that C++11 syntax cannot be used in the editor. How to deal with it?

![image](https://user-images.githubusercontent.com/51160359/179484965-479d09ee-2464-46b1-ae5b-84602a756f7b.png)

**Comment by wotterfree** [comment]
done, tks

**Reply by HighCommander4** [reply]
You need to follow one of the methods described in https://clangd.llvm.org/installation#project-setup to tell clangd about your build flags, including the standard version such as `-std=c++11`.
